### PR TITLE
Master website event issue ajo

### PR DESCRIPTION
**Pull request by psa-odoo**
Issue : https://www.odoo.com/web?#id=643259&view_type=form&model=project.issue&menu_id=4120&action=609

### CODE REVIEW

**Comment by psa-odoo** [comment]
Make the branch single commit.

File: addons/website_event_track/views/event_backend.xml [hunk 1]
```diff
@@ -331,10 +331,14 @@ Tracks define the agenda of your event. These can be a talk, a round table, a me
...
-                        <field name="show_track_proposal"/>
-                        <label for="show_track_proposal"/>
+                        <field name="show_tracks"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <label for="show_tracks"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <field name="show_track_proposal"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
+                        <label for="show_track_proposal"
+                               attrs="{'invisible': [('show_menu', '=', False)]}"/>
```
Comment (psa-odoo) [inline addons/website_event_track/views/event_backend.xml]: Add the group "group_no_one" in 'show_tracks' and 'show_track_proposal' also else when the `Dedicated menu` is marked TRUE and no technical rights are given than these checkboxes will be visible.

File: addons/website_event/models/event.py [hunk 2]
```diff
@@ -56,6 +56,8 @@ def _get_new_menu_pages(self):
     @api.one
     def _set_show_menu(self):
         if self.menu_id and not self.show_menu:
+            self.show_track_proposal = False
+            self.show_tracks = False
```
Comment (atp-odoo) [inline addons/website_event/models/event.py]: this field will be added in website_event_track module.

**Comment by jem-odoo** [comment]
https://github.com/odoo/odoo/commit/b3e762956654c3f34431f6b677add9e2815e6715

**Pull request closed without merging**